kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -117,35 +252,101 @@
   @require '~kolibri.styles.definitions'
   @require '~kolibri.styles.navBarItem'
 
+  $footerheight = 100px
+
   .nav-wrapper
-    display: table
-    table-layout: fixed
+    top: 0px
     background: $core-bg-light
     font-weight: 300
     position: fixed
-    z-index: auto
-    @media screen and (min-width: $portrait-breakpoint + 1)
-      font-size: 1em
-      height: 100%
-      width: $nav-width
-    @media screen and (max-width: $portrait-breakpoint)
-      font-size: 0.8em
-      bottom: 0
-      width: 100%
-      min-width: 300px
+    z-index: 1001
+    font-size: 1em
+    height: 100vh
+    overflow: auto
+    -webkit-overflow-scrolling: touch
+    box-shadow: 2px 0 0px 0px rgba(0, 0, 0, 0.12)
+    .logo
+      margin: auto
+      display: block
 
   .nav-main
     background: $core-bg-light
-    height: 100vh
-    @media screen and (max-width: $portrait-breakpoint)
-      display: table-row
-      height: 56px
 
   a.active:focus svg
     fill: $core-bg-light
 
-  .nav-icon
-    width: 40px
-    height: 40px
+  .header
+    overflow: auto
+    overflow-y: hidden
+    background-color: $core-text-default
+    box-shadow: 0 0 2px rgba(0, 0, 0, 0.12), 0 2px 2px rgba(0, 0, 0, 0.2)
+    .logo, .close
+      float: left
+    .title, .close
+      color: $core-bg-light
+    .close
+      top: 50%
+      transform: translateY(-50%)
+      position: relative
+      border: none
+    .title
+      font-weight: bold
+
+  .footer
+    bottom: 0
+    position: absolute
+    overflow: auto
+    overflow-y: hidden
+    background-color: $core-text-default
+    height: $footerheight
+    .logo, .message-container
+      top: 50%
+      transform: translateY(-50%)
+      position: relative
+    .logo
+      float: left
+    .message-container
+      .message
```

Comment:
recommend avoiding nested selectors:

* http://csswizardry.com/2012/05/keep-your-css-selectors-short/
* http://thesassway.com/beginner/the-inception-rule